File: DataStructures/Stack.java

public class Stack 
{
	int maxSize;
	int top;
	String arr[];
	public Stack(int n) 
	{
		maxSize=n;
		arr=new String[maxSize];
		top=0;
	}
	void Push(String n)
	{
		if(top<maxSize)
		{
			arr[top]=n;
			top++;
		}
		else
		{
			System.out.println("Overflow");
		}
	}
	void Pop()
	{
		if(top>=0)
		{
			top--;
		}
		else
		{
			System.out.println("Underflow");
		}
	}
}

File: DataStructures/Stack/Stack.cpp
#include<iostream>
#include<stack>
using namespace std;
void showStack(stack<int> s)
{
    stack<int> stk=s;
    while(!stk.empty())
    {
        cout<<stk.top()<<" ";
        stk.pop();
    }
}
int main()
{
	stack<int> st;
	st.push(10);
	st.push(20);
	st.push(30);
	st.push(40);
	st.push(50);
	showStack(st);
	cout<<endl;
	cout<<"Size "<<st.size()<<endl;
	cout<<"Top "<<st.top()<<endl;
	
}